source/renderer/app/components/staking/stake-pools/StakePools.js

```diff
@@ -160,50 +187,84 @@ export default class StakePools extends Component<Props, State> {
               search={search}
               onSearch={this.handleSearch}
               onClearSearch={this.handleClearSearch}
+              onGridView={this.handleGridView}
+              onListView={this.handleListView}
+              isListView={isListView}
+              isGridView={isGridView}
+              isFixed={(isFixed || maintainFixed) && !!filteredStakePoolsList.length}
               isClearTooltipOpeningDownward
+              isScrolled={isScrolled}
             />
-
-            {stakePoolsDelegatingList.length > 0 && (
+            {isListView && (
               <Fragment>
-                <h2 className={styles.listTitle}>
-                  {intl.formatMessage(messages.delegatingListTitle)}
+                <h2 className={tableHeadingClasses}>
+                  <FormattedMessage
+                    {...listTitleMessage}
+                    values={{
+                      pools: filteredStakePoolsList.length,
+                    }}
+                  />
                 </h2>
-                <StakePoolsList
-                  listName="stakePoolsDelegatingList"
-                  stakePoolsList={stakePoolsDelegatingList}
+                <StakePoolsTable
+                  listName="selectedIndexTable"
+                  stakePoolsList={filteredStakePoolsList}
                   onOpenExternalLink={onOpenExternalLink}
                   currentTheme={currentTheme}
-                  isListActive={selectedList === 'stakePoolsDelegatingList'}
+                  isListActive={selectedList === 'selectedIndexTable'}
```

Comment:
@DeeJayElly Define constants for string literals instead of using them directly.

Reply:
Fixed